coordinator/app/src/test/kotlin/net/consensys/zkevm/coordinator/app/L1DependentAppTest.kt

```diff
@@ -35,6 +35,7 @@ class L1DependentAppTest {
     val blobsRepository = mock<BlobsRepository>()
     val aggregationsRepository = mock<AggregationsRepository>()
     val lastProcessedBlock = 100uL
+    val lastConsecutiveAggregatedBlockNumber = 80uL
```

Comment:
IMO this test is useless because it's more complicated than the implementation

Reply:
I'd rather keep this as it reflects how the delete record functionss of the batch, blob, aggregation repositories are called.